### ISSUE TITLE
Publish latest changes to npm

### ISSUE THREAD

**Issue by ghost** [stoffern/gl-react-instagramfilters#12]
Please 😄

**Comment by ghost** [comment]
@stoffern I think the package hasn't been published for a year or so 🙏

**Comment by st0ffern** [comment]
Is is not really ment for people to use since it is for educational purposes only.
Its illegal to use it in production 😉 

That is why i have not given it the time to publish on NPM 😉

**Comment by ghost** [comment]
@stoffern yeah, the warning is explanatory enough and it also shows on the npmjs page 😝 somehow it would be cool to be able to use it for educational purpose through the npm registry directly.
